docs/content/en/docs/tutorials/developer-journey.md

```diff
@@ -0,0 +1,159 @@
+```bash
+chmod +x start.sh && ./start.sh
+```
+
+### Start a Minikube cluster
+
+We'll use `minikube` as our local kubernetes cluster of choice.
+
+Run:
+```bash
+minikube start
+```
+
+## Run the App to Minikube Using Skaffold
+
+### Deploy the App
+Run:
+```bash
+skaffold dev
+```
+
+**Important:** note the software versions under the `DETECTING` phase of the buildpack output. These will be important later.
+
+Open a second terminal. In order to connect to the application to make sure its working, you need to start the minikube load balancer by running:
+```bash
+minikube tunnel
+```
+
+Open a third terminal. Find out the load balancer IP by recording the external IP as `EXTERNAL_IP` after running:
+```bash
+kubectl get service
+```
+
+Ensure the app is responding by running:
+```bash
+curl http://EXTERNAL_IP:8080
+```
+
+### Change the App and Trigger a Redeploy
+
+In the Cloud Shell Editor, open `src/main/java/hello/HelloController.java` and change the return line to `return "Hello, Skaffold!"`.
+
+Switch back to the first terminal and see that its rebuilding and redeploying the app.
+
+Switch back to the third terminal and run the following command to watch until only one pod is running:
+```bash
+watch kubectl get pods
+```
+
+Once there is only one pod running, meaning the latest pod is deployed, see that your app changes are live:
+```bash
+curl http://EXTERNAL_IP:8080
+```
+
+### Updating Buildpacks
+
+Perhaps the best benefit of buildpacks is that it reduces how much work developers need to do to patch their applications if the security team highlights a library vulnerability.
+
+In Cloud Shell Editor, open `skaffold.yaml` and update the builder line to `gcr.io/buildpacks/builder:v1`, which will use the latest builder that has more up-to-date buildpacks.
+
+Switch back to the first terminal and see that its rebuilding and redeploying the app.
+
+**IMPORTANT:** compare the software versions under the `DETECTING` phase of the buildpack output to the ones you saw before. The builder is now using newer buildpack versions.
+
+Switch back to the third terminal and run the following command to watch until only one pod is running:
+```bash
+watch kubectl get pods
+```
+
+Once there is only one pod running, meaning the latest pod is deployed, see that your app changes are live:
+```bash
+curl http://EXTERNAL_IP:8080
+```
+
+## Deploy the App to Enterprise GKE Cluster Using Skaffold
+
+### Switch kubectl Context to Enterprise GKE Cluster
+
+Switch your local kubectl context to the enterprise GKE cluster and get the latest credentials:
+```bash
+gcloud container clusters get-credentials skaffold-tutorial-cluster --region us-central1-a
+```
+
+See that kubectl is now configured to use the remote kubernetes cluster instead of minikube (denoted by the asterisk)
+```bash
+kubectl config get-contexts
+```
+
+### Deploy the App
+
+Attempt to deploy the app by running:
+```bash
+skaffold dev --default-repo=gcr.io/skaffold-tutorial
```

Comment:
The project name here should be the users and not hardcoded...

You can do something like

```shell
skaffold dev --default-repo=gcr.io/$(gcloud config get-value project)
```